lib/logstash/filters/aggregate.rb

```diff
@@ -202,13 +215,20 @@ def filter(event)
 			map = aggregate_maps_element.map
 
 			# execute the code to read/update map and event
-			@codeblock.call(event, map)
+			begin
+				@codeblock.call(event, map)
+				noError = true
+			rescue Exception => exception
```

Comment:
OK, no problem for that. I will fix it soon.

Just for information, `rescue Exception => exception` was inspired from this code : 
https://github.com/logstash-plugins/logstash-filter-ruby/blob/master/lib/logstash/filters/ruby.rb#L40

=> Maybe it needs also a fix ?